src/app/Caches/CacheMetrics.tsx

```diff
@@ -222,9 +227,7 @@ const CacheMetrics = (props: { cacheName: string; display: boolean }) => {
           <TextContent>
             <TextList component={TextListVariants.dl}>
               <TextListItem aria-label="view-cache-metrics-off-heap" component={TextListItemVariants.dt}>
-                {displayUtils.formatNumber(
-                  stats.off_heap_memory_used
-                )}
+                { memory==="OFF_HEAP" ? displayUtils.formatNumber(stats.off_heap_memory_used) : "N/A" }
```

Comment:
"N/A" does not sound very meaningful to me. I would prefer something like: "This statistic does not apply to this cache."

Do I have the correct understanding here? If this is something we show to users then I think we need to be clear about what we're saying and "N/A" doesn't cut it imo.

Reply:
Thanks for putting it in context to help me understand @dipguptaredhat 

I'd like to get another opinion on the use of "N/A" because I feel like there might be something more intuitive. I'll respond later today so we can close this off.

Reply:
@dvagnero Please also feel free to make a suggestion on the use of "N/A" here. 

This isn't a phrase we currently use elsewhere in the console (see https://github.com/infinispan/infinispan-console/blob/main/src/app/assets/languages/en.json) and - before we adopt it - I'd like to make sure it's the most meaningful option. There might also be a couple of grammar issues. Strictly speaking "n.a." is the correct abbreviation and the translated form is several characters longer than the English or French form. 

The best alternative I can think of is "null" which might be a winner. What are your thoughts?

Reply:
@oraNod I think I suggested "N/A" first, thinking that column is meant to show only numbers, and it's bolded, so putting a longer description there would mess the layout. "N/A" is already a big step up from displaying "-1" or "0" when the number doesn't make sense but the embedded/REST API still provides it as a number.

Maybe "Not Available" would be clearer while still fitting in the column? Then the full explanation could stay in the tooltip and the layout would stay the same.

Reply:
@danberindei Thanks! Yes, I think the full explanation goes in the tooltip. "Not available" is much more meaningful than "N/A" or "n.a." but loses the compactness and messes with the layout. If we translate to Japanese, even if we use "N/A", we'll hit that same issue.

You're spot on regarding "null". That could indicate an error condition. I think "-" is a good option.

Reply:
Let's go with a dash here instead of "N/A" unless there are objections. I've added this in a commit.